### PR TITLE
Kinetic small fixes

### PR DESCRIPTION
**Pull request by davetcoleman**
Builds on top of https://github.com/RethinkRobotics/baxter_simulator/pull/106

*This PR is sponsored by [Vicarious](http://www.vicarious.com/)*

### CODE REVIEW

File: baxter_gazebo/worlds/baxter.world [hunk 1]
```diff
@@ -8,11 +8,21 @@
...
     </include>
+
     <physics type="ode">
       <real_time_update_rate>1000.0</real_time_update_rate>
       <gravity>
 	0.0 0.0 -9.81
       </gravity>
     </physics>
+
+    <gui fullscreen='0'>
```
Comment (davetcoleman) [inline baxter_gazebo/worlds/baxter.world:19]: starts GUI up better zoomed into Baxter

File: baxter_sim_kinematics/src/arm_kinematics.cpp [hunk 2]
```diff
@@ -79,13 +79,12 @@ bool Kinematics::init_grav()
...
   ros::ServiceClient set_lp_client = nh.serviceClient<gazebo_msgs::SetLinkProperties>("/gazebo/set_link_properties");
 
-  // Wait for service to become availablestd::find(vector.begin(), vector.end(), item)!=vector.end()
+  // Wait for service to become available
   set_lp_client.waitForExistence();
 
   gazebo_msgs::SetLinkProperties setlinkproperties;
   gazebo_msgs::GetLinkProperties getlinkproperties;
 
-  setlinkproperties.request.gravity_mode = 0;
```
Comment (davetcoleman) [inline baxter_sim_kinematics/src/arm_kinematics.cpp]: this line didn't do anything because it is reset below

**Comment by davetcoleman** [comment]
This is the new startup camera angle, much better I think:

screenshot from 2017-02-24 10-19-49

File: baxter_gazebo/worlds/baxter.world (hunk 1 above)
Comment (IanTheEngineer) [reply baxter_gazebo/worlds/baxter.world:19]: I like the new angle :)

File: baxter_sim_kinematics/src/arm_kinematics.cpp (hunk 2 above)
Comment (IanTheEngineer) [reply baxter_sim_kinematics/src/arm_kinematics.cpp]: looks good

**Review by IanTheEngineer — APPROVED** [review]
looks great!

**Pull request merged**